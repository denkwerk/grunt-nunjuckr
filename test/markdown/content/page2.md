# Another test file written in markdown

This is a second test file written in markdown. This file also is a test for grunt-nunjuckr.
