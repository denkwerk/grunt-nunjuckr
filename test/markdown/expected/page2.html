<!DOCTYPE html>
<html>
<head>
    <meta charset="UTF-8">
    <title>Nunjuckr markdown test</title>
</head>
<body>

    <h1 id="anothertestfilewritteninmarkdown">Another test file written in markdown</h1>

<p>This is a second test file written in markdown. This file also is a test for grunt-nunjuckr.</p>

</body>
</html>
